site/docs/develop/kernel.md

```diff
@@ -191,11 +190,11 @@ support atomic CAS (compare-and-swap) operations against a single row/record, bu
 conditional updates to multiple rows/records is either not supported at all or extremely slow.
 
 Nessie differentiates between content types that do require so called _global-state_ and those
-that do not. Apache Iceberg is currently the only content type that supports global state:
-the pointer to the Iceberg "Table Metadata" is tracked as "global state" and 
-the Iceberg snapshot ID, schema ID, partition spec ID, sort order ID
-is tracker per _Nessie named reference_. For _Nessie commits_, which are atomic, this means that
-Nessie has to update both the global-state and the on-reference-state for the Iceberg table. While
+that do not. _Global-state_ is maintained globally and evaluated when a convent value object is
```

Comment:
typo: convent -> content?

Reply:
same here, should we even mentiond global-state here? maybe just remove the entire paragraph?